components/forms/DynamicRow/DynamicRow.tsx

```diff
@@ -113,27 +134,31 @@ export const DynamicGroup = (props: DynamicGroupProps): React.ReactElement => {
               lang={lang}
               t={t}
             />
-            {rows.length > 1 && (
-              <Button
-                type="button"
-                secondary={true}
-                onClick={() => deleteRow(index)}
-                testid={`delete-row-button-${field.name}.${index}`}
-              >
-                {`${lang === "en" ? "Delete" : "Supprimer"} ${rowLabel} ${index + 1}`}
-              </Button>
-            )}
+            <div>
+              {!hasReachedMaxNumberOfRows && index === rows.length - 1 && (
+                <Button
+                  type="button"
+                  secondary={true}
+                  onClick={addRow}
+                  testid={`add-row-button-${field.name}`}
+                >
+                  {`${lang === "en" ? "Add" : "Ajouter"} ${rowLabel}`}
```

Comment:
Might be outside the scope of this PR, but these button labels should be added to a locales json file, instead of being buried in the code.

Reply:
Will create a ticket :)

Reply:
In fact I will deal with it right now because I got time so why not ;)